infrastructure/quick-deploy/aws/all/parameters.tfvars

```diff
@@ -41,35 +41,39 @@ eks = {
 }
 
 # Operational node groups for EKS
-eks_operational_worker_groups = [
-  {
-    name                                     = "operational-worker-ondemand"
-    spot_allocation_strategy                 = "capacity-optimized"
-    override_instance_types                  = ["c5.xlarge"]
-    spot_instance_pools                      = 0
-    asg_min_size                             = 1
-    asg_max_size                             = 5
-    asg_desired_capacity                     = 1
-    on_demand_base_capacity                  = 1
-    on_demand_percentage_above_base_capacity = 100
-    kubelet_extra_args                       = "--node-labels=grid/type=Operator --register-with-taints=grid/type=Operator:NoSchedule"
-  }
-]
+eks_operational_worker_groups = {
+    eks_operational_worker_groups = {
```

Comment:
Why the key is the same as the variable name?

Reply:
node_group was an array now it is a map
for the moment we have cut in 2 maps :
operational worker => eks_operational_worker_groups
workers (armonik) =>eks_worker_groups
As spoken with Lilia next step in the refacto will be to join the 2 maps and we will nomore need the 2 maps.

Reply:
renamed